### PR TITLE
React Native Web integration

### PR DESCRIPTION
**Pull request by azimgd**
Setup RN Web into example project

to be merged first: https://github.com/Expensify/react-native-key-command/pull/8

### CODE REVIEW

**Comment by azimgd** [comment]
@parasharrajat easy one, ready to be reviewed

**Comment by azimgd** [comment]
Next step is to implement web version of the module

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # react-native-key-command
-react-native-key-command
+Objective-C and Java module that registers and listens to specified keyboard events, dispatching the payload to JavaScript.
```
Comment (parasharrajat) [inline README.md:2]: May be

```suggestion
A cross-platform module that registers and listens to specified keyboard events, dispatching the payload to JavaScript.

✅ Android
✅ iOS
✅ Web
```

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-native-key-command",
   "version": "0.1.0",
-  "description": "test",
+  "description": "Objective-C and Java module that registers and listens to specified keyboard events, dispatching the payload to JavaScript",
```
Comment (parasharrajat) [inline package.json:4]:
```suggestion
  "description": "A cross-platform module that registers and listens to specified keyboard events, dispatching the payload to JavaScript",
```

**Review by parasharrajat — CHANGES_REQUESTED** [review]

**Review by parasharrajat — APPROVED** [review]
LGTM.

cc: @luacmartins

**Comment by parasharrajat** [comment]
Oh, there are conflicts.

**Review by parasharrajat — APPROVED** [review]

**Review by luacmartins — APPROVED** [review]

**Pull request merged**